ui/README.md

```diff
@@ -0,0 +1,14 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Local Development using VSCode Chrome Debugger
+
+Run:
+
+### `npm start`
```

Comment:
This is a h3 type heading, I don't think it should be. The `###` should be removed and it should be converted it to a code block like below. See [Code and Syntax Highlighting](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)

```sh
npm start
```